Source/WTF/wtf/Compiler.h

```diff
@@ -43,6 +43,15 @@
 #define COMPILER_HAS_ATTRIBUTE(x) 0
 #endif
 
+/* COMPILER_HAS_ATTRIBUTE() - whether the compiler supports a particular c++ attribute. */
```

Comment:
Given the others in this file, I sort of would've expected `CLANG` to be in the name of this macro. Or maybe the right direction is to remove it from the other similar macros some day.